k8s/base/madara/madara.dockerfile

```
@@ -0,0 +1,53 @@
+# Stage 1: Build the application
+FROM rust:1.81 AS builder
+
+# Install build dependencies
+RUN apt-get -y update && \
```

Hadolint

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)